src/wrappers/themis/react-native-themis/src/index.js

```diff
@@ -236,12 +236,14 @@ export function secureMessageSign64(plaintext, privateKey64, publicKey64) {
     if (!isBase64(privateKey64)) {
         throw new Error("Parameter privateKey64 is not base64 encoded");
     }
+    const privateKey = Array.from(Buffer.from(privateKey64, 'base64'));
+    if (publicKey64 === undefined || publicKey64 === null) {
+        publicKey64 = "";
```

Comment:
why we assign empty string instead of null?

Reply:
Not a relevant comment anymore. :)